dev/install-python-packages.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+pip3 install --upgrade pip
+pip3 install -r opengrok-tools/src/main/python/requirements.txt
```

Comment:
this should not be needed, if you install opengrok-tools with pip the requirements are donwloaded automatically

Reply:
the opengrok-tools package is not installed during build I think

Reply:
It is installed when tests phase is on. So why do you need to install its requirements?

Reply:
because pylint needs the modules and cannot find them

Reply:
aha, ok then

Reply:
Also, I'd like to be able to check the sources without installing the package. After all, the checks precede the install in the pom file.

Reply:
Actually, running pylint from venv is reasonable because it does not clash with system installed site packages.